test/std-in/StdInError.js

```diff
@@ -3,6 +3,7 @@ function hello() {
 }
 function world() {
   return 'world';
+}
```

Comment:
This omission is deliberate. The file is test data, not something that will be executed.